.changeset/purple-clouds-punch.md

```diff
@@ -0,0 +1,5 @@
+---
+"open-next": patch
```

Comment:
Could you remove the changeset for now, i'll add this again before release
The repo will change name very soon and this will cause trouble with changeset